submissions/israelOladosu/php-slim/app/controllers/ArticleController.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace App\Controller;
+
+use App\Model\Article;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+
+session_start();
+
+class ArticleController
+{
+    public function index(Request $request, Response $response)
+    {
+        $articles = Article::all();
+
+        $response->getBody()->write(json_encode(["status" => "success", "data" => [$articles]]));
```

Comment:
Remove the square bracket. the `data` value should be an `object` not an `array`. The same goes for other places

Reply:
Done now. Thank you